package-lock.json

```diff
@@ -1,8 +1,350 @@
 {
   "name": "stats.js",
   "version": "0.17.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
I've used npm 8.1.0, please let me know if you would want me to use an older version.